staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/envelope.go

```diff
@@ -189,28 +178,11 @@ func (t *envelopeTransformer) addTransformer(encKey []byte, key []byte) (value.T
 		return nil, err
 	}
 	transformer := t.baseTransformerFunc(block)
-	// Use base64 of encKey as the key into the cache because hashicorp/golang-lru
-	// cannot hash []uint8.
-	if t.cacheEnabled {
-		t.transformers.Add(base64.StdEncoding.EncodeToString(encKey), transformer)
-		metrics.RecordDekCacheFillPercent(float64(t.transformers.Len()) / float64(t.cacheSize))
-	}
+	// TODO(aramase): Add metrics for cache fill percentage with custom cache implementation.
```

Comment:
Please create an issue for this (can be the same issue that tracks the new cache implementation work).  And for adding `TestEnvelopeCacheLimit` back (if it makes sense for the new cache).

Reply:
opened https://github.com/kubernetes/kubernetes/issues/115275